pkgs/top-level/all-packages.nix

```diff
@@ -19636,6 +19636,7 @@ in
     postgresql_11
```

Comment:
Maybe we want to drop some older version(s)?

Reply:
All versions back to 9.6 inclusive are still supported upstream, so they cost us almost nothing to maintain. And a lot of people run production systems on older db releases as it's hard to schedule the downtime for an upgrade.

Reply:
@risicle is right. 
These versions are still widely used in production, we can't drop them so lightly